File: flutter/lib/models/pakaian.dart
class Pakaian {
  int id;
  String nama;
  String deskripsi;
  String kategori;
  String kelamin;
  String usia;
  String status;
  int harga;
  String image;

  Pakaian({
    required this.id,
    required this.nama,
    required this.deskripsi,
    required this.kategori,
    required this.kelamin,
    required this.usia,
    required this.status,
    required this.harga,
    required this.image,
  });

  factory Pakaian.fromJson(Map<String, dynamic> json) {
    return Pakaian(
      id: json['id'],
      nama: json['nama'],
      deskripsi: json['deskripsi'],
      kategori: json['kategori'],
      kelamin: json['kelamin'],
      usia: json['usia'],
      status: json['status'],
      harga: json['harga'],
      image: json['image'],
    );
  }
}

File: laravel/app/Models/Pakaian.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Model;
use Illuminate\Database\Eloquent\SoftDeletes;

class Pakaian extends Model
{
    use SoftDeletes;

    protected $fillable = ['nama', 'image', 'deskripsi', 'kategori', 'kelamin', 'usia', 'status', 'harga'];

    protected $dates = ['deleted_at'];
}
